Dockerfile.dapper594236427

```diff
@@ -0,0 +1,17 @@
+FROM quay.io/submariner/shipyard-dapper-base
```

Comment:
This file shouldn't be added. IIRC, this is just used for building image to run e2e.

Reply:
Oh it was a side effect of rebase and conflict resolution, I will remove it.